js/background.js

```diff
@@ -69,6 +69,16 @@ function Background() {
         });
       }
     });
+
+    if (os !== 'w') {
+      chrome.tabs.create({ url: 'chrome://settings/searchEngines' });
+      chrome.notifications.create({
+        type: "basic",
```

Comment:
@bsstoner This is indeed very basic and might not work as well as we would like it to.

It might be interesting to add some [Rich notifications](https://developer.chrome.com/extensions/richNotifications), especially with some nice pictures which will make the whole process easier for everyone.